target/linux/ramips/image/mt76x8.mk

```diff
@@ -189,6 +189,25 @@ define Device/mercury_mac1200r-v2
 endef
 TARGET_DEVICES += mercury_mac1200r-v2
 
+define Device/netgear_r6020
+  BLOCKSIZE := 64k
+  IMAGE_SIZE := 7104k
+  DEVICE_VENDOR := NETGEAR
+  DEVICE_MODEL := R6020
+  DEVICE_PACKAGES := kmod-mt76x2 kmod-usb2 kmod-usb-ohci
```

Comment:
Please decide whether you have USB or not. Currently, you disable it in DTS.

Reply:
This was copied from another similar device.  I'll remove it and try again.

Reply:
So, do you have USB or not?

Reply:
No.  This device does not have USB ports.